manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/cache.yaml

```diff
@@ -205,7 +205,7 @@ spec:
       - name: server
         image: {{ .Values.images.cacheserver }}
         env:
-          {{ if .Values.managedstorage.enabled }}
+          {{ if .Values.managedstorage.cloudsqlInstanceConnectionName }}
```

Comment:
Should we check both `.Values.managedstorage.enabled ` and `cloudsqlInstanceConnectionName` for enabling cloudsql instance?

Reply:
I wonder about this...
If the user specifies a connection, it's a bit confusing when it's not used.
Ideally we should make the cloudsqlInstanceConnectionName input enabled only when the managedstorage is set to true.

Reply:
Do we have a mechanism to make sure `enabled` and `cloudsqlInstanceConnectionName` are exclusive to each other? So we have the confidence that user won't set both variables to be conflicting values. Sorry that I don't have much context on how marketplace manifest is provisioned, but it is good to make sure both parameters are checked, if we don't have anywhere else to enforce the input is valid.